independent-projects/tools/analytics-common/src/main/java/io/quarkus/analytics/ConfigService.java

```diff
@@ -36,7 +36,7 @@ public class ConfigService {
             + "--- Help improve Quarkus ---" + NEW_LINE
             + "----------------------------" + NEW_LINE
             + "* Learn more: https://quarkus.io/usage/" + NEW_LINE
-            + "* Do you agree to contribute anonymous build time data to the Quarkus community? (y/n) " + NEW_LINE;
+            + "* Do you agree to contribute anonymous build time data to the Quarkus community? (y/n and then enter) " + NEW_LINE;
```

Comment:
```suggestion
            + "* Do you agree to contribute anonymous build time data to the Quarkus community? (y/n + enter) " + NEW_LINE;
```

just to avoid too long line?

Reply:
I can see some people reaching for the `+` key :)

Reply:
I propose  (y/n and enter)